test/example.md

```diff
@@ -1,3 +1,39 @@
+- [Interface example](#interface-example)
```

Comment:
Can we somehow limit the level of entries that get pulled into the Contents? (i.e. say only 1st and 2nd level entries are included in the contents)
I think we can add this as an optional parameter to `--contents` option.